admin/partials/settings/compressor.php

```diff
@@ -17,36 +17,48 @@
 
 defined( 'WPINC' ) || die;
 
-ob_start();
-
-$selected = 'selected="selected"';
-
-$available_compressors = array( 'php_zip', 'pcl_zip' );
-
-// Settings for compressor.
-$pcl_zip_selected = empty( $settings['compressor'] ) || 'pcl_zip' === $settings['compressor'] ?
-	$selected : '';
-
-$php_zip_selected = ! empty( $settings['compressor'] ) && 'php_zip' === $settings['compressor'] ?
-	$selected : '';
+$core = apply_filters( 'boldgrid_backup_get_core', null );
 
-$php_zip_option = ! in_array( 'php_zip', $available_compressors, true ) ?
-	'' : sprintf( '<option value="php_zip" %1$s>ZipArchive</option>', $php_zip_selected );
+// An array of compressors we will let the user choose from.
```

Comment:
You may want to change the order of the list so that the compressors are listed in a preferred order.  For example: "System zip", "ZipArchive", "PclZip".

Reply:
By default, the default compressor will be selected.

We can make this change when system_zip becomes official.

Right now, just trying to release...